src/job-execution/WebJobExecutor.ts

```diff
@@ -144,10 +145,23 @@ export class WebJobExecutor extends BaseJobExecutor {
     const requestPromise = new Promise((resolve, reject) => {
       this.requestClient!.post(apiUrl, formData, undefined)
         .then(async (res: any) => {
-          this.requestClient!.appendRequest(res, sasJob, config.debug)
-
           let jsonResponse = res.result
 
+          if (this.serverType === ServerType.Sasjs && config.debug) {
+            this.requestClient!.appendRequest(
+              jsonResponse,
+              sasJob,
+              config.debug
+            )
+            jsonResponse = JSON.parse(parseWeboutResponse(jsonResponse._webout))
```

Comment:
`JSON.parse` throws an exception.
And shouldn't `parseWeboutResponse` function return a parsed value as the name suggests?